### ISSUE TITLE
tests.stockfishchess.org/signup - HTTPS

### ISSUE THREAD

**Issue by Zyano** [glinscott/fishtest#335]
Hello,

Any chance for https support on tests.stockfishchess.org. This could possibly be done using Lets Encrypt.

Best regards

**Comment by ppigazzini** [comment]
Already done on the development server using nginx as reverse proxy. Anyway the official server will be updated to https only after have iron out some wrinkles (e.g. the buttons to show/hide the machines and the queue of pending tests don't work with https).

**Comment by Zyano** [comment]
Fair enough. I was just looking to create a user for the test platform. I have some CPU power that could be contributed to the test effort. In any case thanks for the reply and hope you get it working :)

**Comment by tomtor** [comment]
@ppigazzini Hi, I would like to have a look at the "Show" button issue. Please let me know when it is a convenient time that I can access the test server via https/reverse-proxy...

**Comment by ppigazzini** [comment]
@tomtor server up and running (ubuntu18.04)
http://dfts-0.pigazzini.it

Reverse proxy without SSL seems running OK
Reverse proxy with SSL has several issues:
 - 'Show' buttons not working
 - 'New' form shows all the parameter boxes (SPRT, SPSA, Fixed)

Reverse proxy with nginx following the [pyramid guide](https://docs.pylonsproject.org/projects/pyramid-cookbook/en/latest/deployment/nginx.html)

I wrote this reverse-proxy.conf because is not working for me the configuration suggested by pyramid (certbot wrote the reverse-proxy-ssl.conf using this as template when installing let's encrypt) 

```
server  {
  listen  80;
  listen [::]:80;
  server_name dfts-0.pigazzini.it;
  location / {
    proxy_set_header X-Forwarded-Proto $scheme;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Host $host:$server_port;
    proxy_set_header X-Forwarded-Port $server_port;

    client_max_body_size    10m;
    client_body_buffer_size 128k;
    proxy_connect_timeout   60s;
    proxy_send_timeout      90s;
    proxy_read_timeout      90s;
    proxy_buffering         off;
    proxy_temp_file_write_size 64k;

    proxy_pass http://127.0.0.1:6543;
    proxy_redirect          off;
  }
}
```

This the snippet of production.ini
```
###
# wsgi server configuration
###

[server:main]
use = egg:waitress#main
host = 0.0.0.0
port = 6543
#listen = *:6543
#host = 127.0.0.1
#port = %(http_port)s

trusted_proxy = 127.0.0.1
trusted_proxy_count = 1
trusted_proxy_headers = x-forwarded-for x-forwarded-host x-forwarded-proto x-forwarded-port
clear_untrusted_proxy_headers = yes
```
